app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Action.java

```diff
@@ -23,6 +23,15 @@
     @SerializedName("query")
     @Expose
     private String query;
+    @SerializedName("latitude")
+    @Expose
+    private String latitude;
+    @SerializedName("longitude")
+    @Expose
+    private String longitude;
+    @SerializedName("zoom")
+    @Expose
+    private String zoom;
```

Comment:
Yup, they need to be changed. make separate POJO classes for each action type and use inheritance  I am creating a separate issue for it.